plane/src/drone/command.rs

```diff
@@ -34,6 +34,10 @@ pub struct DroneOpts {
     #[clap(long)]
     docker_runtime: Option<String>,
 
+    // Optional path to a unix socket for connecting to an external executor.
```

Comment:
If you add a third `/`, this comment will be parsed as a docstring and be printed when running `plane --help`